index.html

```diff
@@ -0,0 +1,118 @@
+			</div>
+			<div class="card-body">
+				<p class="card-text">
+					Become a successful software developer. You can apply for Microverse from any country in the world.
+          Take the first step towards your global career today.
+				</p>
+			</div>
+			<div class="card-footer">
+				<p class="date">2022.12.01(THU) ~ 16(FRI)</p>
+				<p class="venue">&copy; Microverse, Get ready to learn everything</p>
+			</div>
+		</main>
+		<section class="main-program flex-column">
+			<h4 class="program-heading">Main<span> Pro</span>gram</h4>
+			<article class="main-program-items">
+				<div class="program-item">
+					<object data="./img/languages/html.svg"></object>
+					<h5>HTML</h5>
+					<p>
+						The HyperText Markup Language or HTML is the standard markup language for documents designed to be displayed in a web browser.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/css.svg"></object>
+					<h5>CSS</h5>
+					<p>
+						Cascading Style Sheets or CSS is a style sheet language used for describing the presentation of a document written in a markup language.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/javascript.svg"></object>
+					<h5>JavaScript</h5>
+					<p>
+						JavaScript (JS) is a language that enables you to create dynamically updating content, control multimedia.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="img/languages/ruby.svg"></object>
+					<h5>Ruby</h5>
+					<p>
+						Ruby is an interpreted, high-level, general-purpose programming
+                language which supports multiple programming.
+					</p>
+				</div>
+				<div class="program-item">
+					<object data="/img/languages/github.svg"></object>
+					<h5>GitHub</h5>
+					<p>
+						GitHub, is an Internet hosting service for software development and version control using Git.
+					</p>
+				</div>
+			</article>
+			<footer class="main-footer flex-column">
+				<a href="" class="main-footer-desktop">SEE THE WHOLE PROGRAM</a>
+				<button class="main-footer-mobile">
+					<p href="#">Join<br>&copy; Microverse<br>2022</p>
+				</button>
+			</footer>
+		</section>
+		<!--dynamicaly feature speakers  -->
+		<section class="speakers flex-column">
+			<h4 class="speakers-heading">Featur<span>ed s</span>peakers</h4>
+			<article class="speakers-items">
+			</article>
+		</section>
+		<section class="partners flex-column">
+			<h4 class="partners-heading">P<span>artn</span>ers</h4>
+			<ul class="partners-body flex-row">
+				<li><img src="/img/partners/PIA.png" alt="PIA's Icon"></li>
+				<li><img src="/img/partners/Google.png" alt="Google's Icon"></li>
+				<li><img src="/img/partners/Mozilla.png" alt="Mozilla's Icon"></li>
+			</ul>
+		</section>
+		<footer class="footer flex-row">
+			<a href=""><img src="/img/Microverse.jpg" alt=""></a>
+			<p> Join Microverse 2022.</p>
+		</footer>
```

Comment:
- [ ] Please make your footer look similar to that of the original design.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/192003517-000b3a6e-3a03-4457-8dfd-67c20ec0adc2.png)

### Required
![image](https://user-images.githubusercontent.com/45460192/192003421-fd883197-bd73-4e27-87de-cfbc6b40f7da.png)